### PR TITLE
Update Dockerfile for 0.2.x version #20

### PR DESCRIPTION
**Pull request by pkruk**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -6,8 +6,16 @@ RUN	    apk update && \
 		    py3-feedparser \
 		    py3-tornado \
 		    py3-yaml \
+			py3-pip \
+			gcc \
```
Comment (pbui) [inline Dockerfile]: Hmm... are the requirements not packaged in Alpine?  I'd rather not have to import a whole compiler into the docker image just to build any Python packages.

If necessary, we can switch the base image to Ubuntu.

Comment (pkruk) [reply Dockerfile]: Well basically from what I see, dateutil requires a compilation of few c modules which include Python.h etc. 
Unfortunately, this reqs are not added to the default alpine image.

Comment (pkruk) [reply Dockerfile]: If you like we can try with python3 docker image :)

Comment (pbui) [reply Dockerfile]: Sure, let's try the python3 docker image.

Comment (pkruk) [reply Dockerfile]: done we want to have a figlet in image? add it?

Comment (pbui) [reply Dockerfile]: It could be optional... I don't think it is critical

Comment (pkruk) [reply Dockerfile]: Ok I added figlet, run docker, run unit tests:

```
# python -m unittest discover -s tests/
.........
----------------------------------------------------------------------
Ran 9 tests in 0.037s

OK
# 
```
Looks fine :)

**Comment by pbui** [comment]
@pkruk Thank you for your contribution

**Pull request merged**